products/views.py

```diff
@@ -0,0 +1,161 @@
+from django.http import HttpResponse, JsonResponse
+from django.views import View
+from .models import  *
+from attributes.models import *
+import json
+
+class BagView(View):
+    def get(self, request):
+
+        total_bag_info = []
+        bag_list = Product.objects.filter(
+                                            name__contains="CHANEL 19"
+                                         ).prefetch_related(
+                                                            'texture',
+                                                            'productimage_set'
+                                                           ).exclude(id__lt=400)
```

Comment:
줄바꿔서 정리는 잘해주셨는데요 마찬가지로 들여쓰기 정리 부탁 드려요.
```suggestion
                   name__contains="CHANEL 19"
            ).prefetch_related(
                  'texture',
                  'productimage_set'
          ).exclude(id__lt=400)
```